### ISSUE TITLE
Console init failure when no config exist

### ISSUE THREAD

**Issue by mnapoli** [piwik/piwik#7968]
When no config file exist (in the Plugin Manager):

```php
$plugins = @Config::getInstance()->Plugins['Plugins'];
// $plugins is null
$this->makePluginsToLoad($plugins);
```

```php
// null is passed in the parameter, the catchable fatal error is somehow silenced because suppressed
private function makePluginsToLoad(array $pluginsToLoad)
{
    $pluginsToLoad = array_unique($pluginsToLoad);
    // ... $pluginsToLoad stays null…
    return $pluginsToLoad; // returns NULL
}
```

This affects the loading of DI config (and maybe other things?): no plugin DI config is loaded. Additionally even when this is fixed, the Diagnostics plugin should always be loaded because we might want to run the systemcheck from the CLI before installing Piwik (which wasn't possible or documented, except that's how it's done in EnterpriseAdmin plugin so that's why it broke).

Given the plugin manager has been refactored for 2.14 I think the bug doesn't exist on master. However 2.13.1 is affected.

**Comment by mnapoli** [comment]
This is fixed in 6928474 in the [`version/2.13.2` branch](https://github.com/piwik/piwik/tree/version/2.13.2), we'll probably need to tag a new release from that.

**Comment by mnapoli** [comment]
It seems the patch will be applied manually by the PRO team.
